server/router/api.go

```diff
@@ -24,168 +24,172 @@ import (
+			queue.GET("/pause", api.PauseQueue)
+			queue.GET("/resume", api.ResumeQueue)
+			queue.GET("/norunningpipelines", api.BlockTilQueueHasRunningItem)
+		}
 
-	secrets := e.Group("/api/secrets")
-	{
-		secrets.Use(session.MustAdmin())
-		secrets.GET("", api.GetGlobalSecretList)
-		secrets.POST("", api.PostGlobalSecret)
-		secrets.GET("/:secret", api.GetGlobalSecret)
-		secrets.PATCH("/:secret", api.PatchGlobalSecret)
-		secrets.DELETE("/:secret", api.DeleteGlobalSecret)
-	}
+		secrets := apiBase.Group("/secrets")
+		{
+			secrets.Use(session.MustAdmin())
+			secrets.GET("", api.GetGlobalSecretList)
+			secrets.POST("", api.PostGlobalSecret)
+			secrets.GET("/:secret", api.GetGlobalSecret)
+			secrets.PATCH("/:secret", api.PatchGlobalSecret)
+			secrets.DELETE("/:secret", api.DeleteGlobalSecret)
+		}
 
-	if zerolog.GlobalLevel() <= zerolog.DebugLevel {
-		debugger := e.Group("/api/debug")
+		logLevel := apiBase.Group("/log-level")
 		{
-			debugger.Use(session.MustAdmin())
-			debugger.GET("/pprof/", debug.IndexHandler())
-			debugger.GET("/pprof/heap", debug.HeapHandler())
-			debugger.GET("/pprof/goroutine", debug.GoroutineHandler())
-			debugger.GET("/pprof/block", debug.BlockHandler())
-			debugger.GET("/pprof/threadcreate", debug.ThreadCreateHandler())
-			debugger.GET("/pprof/cmdline", debug.CmdlineHandler())
-			debugger.GET("/pprof/profile", debug.ProfileHandler())
-			debugger.GET("/pprof/symbol", debug.SymbolHandler())
-			debugger.POST("/pprof/symbol", debug.SymbolHandler())
-			debugger.GET("/pprof/trace", debug.TraceHandler())
+			logLevel.Use(session.MustAdmin())
+			logLevel.GET("", api.LogLevel)
+			logLevel.POST("", api.SetLogLevel)
 		}
-	}
 
-	logLevel := e.Group("/api/log-level")
-	{
-		logLevel.Use(session.MustAdmin())
-		logLevel.GET("", api.LogLevel)
-		logLevel.POST("", api.SetLogLevel)
+		apiBase.GET("/signature/public-key", session.MustUser(), api.GetSignaturePublicKey)
+
+		apiBase.POST("/hook", api.PostHook)
+
+		if zerolog.GlobalLevel() <= zerolog.DebugLevel {
+			debugger := apiBase.Group("/debug")
+			{
+				debugger.Use(session.MustAdmin())
+				debugger.GET("/pprof/", debug.IndexHandler())
+				debugger.GET("/pprof/heap", debug.HeapHandler())
+				debugger.GET("/pprof/goroutine", debug.GoroutineHandler())
+				debugger.GET("/pprof/block", debug.BlockHandler())
+				debugger.GET("/pprof/threadcreate", debug.ThreadCreateHandler())
+				debugger.GET("/pprof/cmdline", debug.CmdlineHandler())
+				debugger.GET("/pprof/profile", debug.ProfileHandler())
+				debugger.GET("/pprof/symbol", debug.SymbolHandler())
+				debugger.POST("/pprof/symbol", debug.SymbolHandler())
+				debugger.GET("/pprof/trace", debug.TraceHandler())
+			}
+		}
 	}
 
-	e.GET("/api/signature/public-key", session.MustUser(), api.GetSignaturePublicKey)
-
 	// TODO: remove /hook in favor of /api/hook
 	e.POST("/hook", api.PostHook)
-	e.POST("/api/hook", api.PostHook)
 
 	// TODO: move to /api/stream
 	sse := e.Group("/stream")
```

Comment:
We should simply rename those streams as well. As they are not documented we just need to make sure the frontend uses the new path.

Reply:
something for another pull